content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+|**Field**|**Value**|
+|---|---|
+|Name|*pack-tutorial-profile*|
+|Version|`1.0.0`|
+|Description|*Cluster profile as part of the pack tutorial.*|
+|Type|*Full*|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:true`|
+
+<br />
+
+### Section: Cloud Type
+In the next section, **Cloud Type**, choose AWS as the infrastructure provider for this tutorial (for a consistent experience). If you choose any other cloud service provider, the core infrastructure layers options, as outlined in the table below, will differ from this tutorial. 
+
+<br />
+
+### Section: Profile Layers
+In the **Profile Layers** section, add the following core infrastructure layers:
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|OS|Public Repo|Ubuntu|`LTS__20.4.x`|
+|Kubernetes|Public Repo|Kubernetes|`1.24.x`|
+|Network|Public Repo|Calico|`3.25.x`|
+|Storage|Public Repo|Amazon EBS CSI|`1.16.x`|
+
+Once you confirm adding the core infrastructure layers, click the **Add New Pack** button to add the following layers (including the custom add-on pack you defined and pushed to the registry). 
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|Authentication | Public Repo| Spectro Proxy | `1.3.x`|
+| App Services | **private-pack-registry** | **Hello Universe** | `1.0.x` |
+
+Note that the values, **private-pack-registry** and the **Hello Universe**, may vary if you have chosen a different name for your registry and pack, respectively. 
+
+
+<br />
+
+![Visual of the Profile Layers success.](/tutorials/create-pack/registries-and-packs_create-pack_profile-layer.png)
+
+<br /> <br />
+
+
+### Section: Review
+Lastly, review and finish creating the profile configuration. You would see the newly created profile if there were no errors or compatibility issues between the individual layers. 
+
+<br /> <br />
+
+## Create a Cluster
+View the details of the profile you just created, and click the **Deploy** button on the top right to create a new cluster. The cluster creation wizard will present you with the a few sections. 
+<br />
+
+### Section: Basic Information
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Cluster name| *pack-tutorial-cluster* |
+|Description| *Cluster as part of the pack tutorial.*|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:true`|
+|Cloud Account|**spectro-cloud**|
+
+Note that the value **spectro-cloud** is the AWS cloud account name already configured in the Palette project settings. 
+
+<br /> <br />
+
+### Section: Parameters
+Next, the **Parameters** section allows you to change the profile configurations. For example, clicking on the **Hello Universe 1.0.x** layer will allow you to configure the `registry`, `repository`, and `tag` parameters defined in the **values.yaml** file. Let's keep those parameter values as default and move to the next section. 
```

Comment:
```suggestion
Next, the **Parameters** section allows you to change the profile configurations. For example, clicking on the **Hello Universe 1.0.x** layer will allow you to configure the `registry`, `repository`, and `tag` parameters defined in the **values.yaml** file. Keep those parameter values as default and move to the next section. 
```

You should add your image from earlier here. This is a better place for it than how it is used earlier in the tutorial.